sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClient.cs

```diff
@@ -209,5 +209,15 @@ public virtual GenericResourceOperations GetGenericResourceOperations(string id)
 
             return new GenericResourceOperations(DefaultSubscription, id);
         }
+
+        /// <summary>
+        /// Gets the RestApi definition for a given Azure namespace.
+        /// </summary>
+        /// <param name="nameSpace"> The namespace to get the rest API for. </param>
+        /// <returns> A container representing the rest apis for the namespace. </returns>
+        public virtual RestApiContainer GetRestApis(string nameSpace)
```

Comment:
Have an override to allow user to specify api-version for the given namespace

Reply:
Since api is tied to resource level it might get very confusing to be able to set the api version at this level.  For preview i would like to keep this as is.